import test_multi_error

me2 = test_multi_error.multi_error2()
print(str(me2))
